docs/guide/quickstart.rst

```diff
@@ -0,0 +1,295 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_quickstart:
+
+Quickstart
+===========
+
+This page will contain a simple explanation of how to make a basic bot using the commands extension "ext.commands".
+Before you begin, make sure to install discord.py - see :ref:`guide_intro_installation` for more information.
+
+Obtaining a Bot Token
+----------------------
+
+You will need a bot application before being able to run a bot. A step-by-step tutorial with screenshots follows:
+
+1. Go to the `Discord Developer Portal <https://discord.com/developers/applications/>`_.
+
+2. Create a new application if you have not made one already:
+    .. image:: /images/discord_create_app_button.png
+        :scale: 90 %
+
+3. Navigate over to the ``bot`` tab:
+    .. image:: /images/discord_bot_tab.png
+        :scale: 80%
+
+4. Create a new Bot user if you have not done so already:
+    .. image:: /images/discord_create_bot_user.png
+
+5. If you need these intents, scroll down and click on these check boxes:
```

Comment:
Assuming these are beginners to discord bots, they likely won't know what intents they need. Perhaps this should link to the intents guide, once created.